docgen/src/guides/using-multiple-instantsearch.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Using multiple InstantSearch instances
+layout: guide.pug
+category: guide
+navWeight: 0
+---
+
+Having several instances of `InstantSearch` can be useful when: 
+
+* You want to make a search on multiple indices 
+* You want to synchronize multiple `InstantSearch` instances
+
+To be able to do that, we are going to take advantage of two props on the [InstantSearch root component](/component/InstantSearch.html).
+
+* onStateChange(nextState): a function being called every time the `InstantSearch` state is updated. 
+* state: an object that is the current state of InstantSearch
+
+The idea is to have a main component that will receive every new state of the first instance and then pass it back to each `InstantSearch` instances.  
+
+Be aware that just having a refinement inside the `InstantSearch` state doesn't apply it automatically to the search. 
```

Comment:
Would not use "be aware" since the user does not know it first.

"Refinements and parameters of an `InstantSearch` state needs to have corresponding widgets, or VirtualWidgets, added to be effectively applied".